addons/block_code/ui/blocks/utilities/template_editor/template_editor.gd

```diff
@@ -114,13 +114,11 @@ func _append_input_parameter(parameter: Dictionary, id: int):
 	parameter_input.placeholder = parameter["name"]
 	parameter_input.variant_type = parameter["type"]
 
-	if parameter["is_option"] and default_value is OptionData:
```

Comment:
FWIW, `is_option` is never set in the current PR. I guess it must have existed in an earlier version.